src/scope.md

```diff
@@ -0,0 +1,132 @@
+# Scope
+
+## Scope of the Reference
+
+The scope of the Rust Reference is to fully describe the syntax and semantics of the Rust language. In this context, the language encompasses:
+
+- Valid syntax and forms.
+- Meaning and interpretation of the syntax.
+- Semantic properties of the runtime behavior.
```

Comment:
What exactly is meant by "Semantic properties of the runtime behavior"? This sounds vaguely opsem-related and yet I have no idea what you mean. ;)

FWIW, in a typical PL textbook, we'd typically have 3 stages for defining a language; I am not sure if the first three items are meant to match that.
- Formal grammar (corresponds to "valid syntax and forms" I think)
- Static semantics -- this is the type system and all the other static checks the compiler might be doing (in Rust that would include borrow checking [unless you consider that to be part of the type system], const checking, things like that)
- Dynamic semantics -- the set of legal "observable behaviors" or "outcomes" or whatever you want to call it of any program